src/pages/Login/Login.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import logo from '../../images/Logo.png';
+import logo_wecode from '../../images/logo_wecode.png';
+// images 폴더의 위치 차이 (public, src)
+import './Login.scss';
+import { useNavigate } from 'react-router-dom';
+
+const Login = () => {
+  const navigate = useNavigate();
+  const goToMain = () => {
+    navigate('./main');
```

Comment:
실제 Router에 path로 작성한 경로로 이동하여야 합니다.

```suggestion
    navigate('/main');
```